package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "telemetry-events",
-  "version": "1.0.1",
+  "version": "1.1.0",
```

Comment:
I was thinking more along the lines of es6-compatible. It can now do things it could not do before, in a backwards compatible manner. The dependency `clone` bumped a major version.